sftp/1.4/modules/ROOT/pages/sftp-connector-troubleshooting.adoc

```diff
@@ -0,0 +1,99 @@
+= Troubleshoot SFTP Connector - Mule 4
+
+To troubleshoot Anypoint Connector for SFTP (SFTP Connector), become familiar with the information about troubleshooting connectivity, permissions and server-side issues, troubleshooting Mule application issues, and interpreting commonly thrown messages.
```

Comment:
remove "troubleshooting" from "troubleshooting Mule application issues," since the first "troubleshooting" already qualifies this.